modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -178,6 +183,9 @@ server.jvm.additional=--add-modules jdk.incubator.vector
 
 To list all vector indexes in a database, use the `SHOW VECTOR INDEXES` command:
```

Comment:
what readers will get from knowing that it is the same command - they are interested in creating a vector index here, the awareness of it being a similar command to create, e.g, a point index seems redundant.

Reply:
I think it's more todo with the view of it that we have (at least previously) tried to put out: the type filtering is just syntax sugar for `WHERE type = 'VECTOR'` and not actually a  separate command. so mentioning that this the same command with filtering on vector indexes keeps that view up instead of "helping" users see them as two separate commands (and start wanting special handling for each index type)

which is also why we have the same set of columns regardless of which type you filter on (and not for example adding columns for dimension or similarity function to `SHOW VECTOR INDEXES` as it is just the `SHOW INDEXES` command with filtering, we did recently get some slack questions around this)

Reply:
So it's not that there is a similar command, but that it is just part of the same command 🤷